lec11-model-selection.Rmd

```diff
@@ -0,0 +1,256 @@
+
+## Enter Information Theory and model selection criteria
+
+There are numerous model selection criteria based on mathematical information theory that we can use to select models among a set of candidate models. They additionally allow the relative weights of different models to be compared and allow multiple models to be used for inferences. The most commonly used information criteria in ecology and evolution are: Akaike's Information Criterion (AIC), the corrected AIC~c~ (corrected for small sample sizes), and the Bayesian Information Criterion (BIC, also known as the Schwarz Criterion)(Johnson and Omland, 2004). Here we will focus on AIC and AIC~c~. Here is how AIC is calculated:
+
+$$ AIC = -2Log\mathcal{L} \ + \ 2p $$
+The lower the AIC value, the better the model. $-2Log\mathcal{L}$ is called the **negative log likelihood** of the model, and measures the model's fit (or lack thereof) to the observed data: **Lower** negative log-likelihood values indicate a beter fit of the model to the observed data. $2p$ is a _bias correcting factor_ that penalizes the model AIC based on the number of parameters (p) in the model. 
+
+Similar to AIC is AIC~c~, which corrects for small sample sizes. It is recommended to use AIC~c~ when $n/k$ is less than 40, with $n$ being the sample size (i.e. total number of observations) and $k$ being the number of parameters in the most saturated model (i.e. the model with the most parameters)(Symonds and Moussalli 2011). AIC~c~ is calculated as follows:
+
+$$ AIC_c = AIC+\frac{2k(k+1)}{n-k-1} $$
+
+## Example of model selection
+
+In lecture 8, we used mixed-effect models to determine whether species richness was influenced by NAP (i.e. the height of a sampling location relative to mean tidal level) across 9 beaches. We generated 3 models: (1) A random intercept model with NAP as a fixed effect and the random effect allowing the intercept (i.e species richness) to vary by beach; (2) A random intercept and slope model with NAP as a fixed effect and a random effect allowing both the intercept (i.e. richness) and slope (i.e. response of richness to NAP) to vary across beaches; and (3) An intercept only model with no fixed effects but allowing for variation in richness across beaches. In this lecture, we'll create similar models with random intercepts and random intercepts + slopes but we'll additionally include Exposure and the NAP by Exposure interaction as additional fixed-effects. Let's load in the RIKZ data.
+
+```{r}
+# Load in packages used throughout lesson
+library(tidyverse)
+library(lme4)
+library(lmerTest)
+library(MuMIn)
+```
+
+```{r}
+# Load in data
+rikz_data <- paste0("https://raw.githubusercontent.com/James-S-Santangelo/",
```

Comment:
Probably should fix this now so the repo is self-contained (maybe easier in the future to comment out the download too.)